### PR TITLE
update log argument name

### PR DESCRIPTION
**Pull request by mallexxx**
Task/Issue URL: https://app.asana.com/0/414235014887631/1204336333309995/f
macOS PR: https://github.com/duckduckgo/macos-browser/pull/1102
BSK PR: https://github.com/duckduckgo/BrowserServicesKit/pull/307

Description:
- standardize log arguments naming

Testing Steps:
validate build works

### CODE REVIEW

**Review by graeme — APPROVED** [review]
I haven’t managed to test this on iOS because of the Xcode auto-update (which I will need to revert). It seems minimal so I’m happy to approve but maybe ask someone else if you want another pair of eyes.

**Review by Bunn — APPROVED** [review]
LGTM! Logging is working

**Pull request merged**
